src/box/memtx_tx.h

```diff
@@ -557,6 +609,9 @@ memtx_tx_snapshot_clarify_slow(struct memtx_tx_snapshot_cleaner *cleaner,
 /**
  * Like a common clarify that function returns proper tuple if original
  * tuple in index is dirty.
+ *
```

Comment:
Since we removed garbage collection in this method we should remove the comment as well.

Reply:
Sorry, wasn't careful enough, fixed.